modules/twinkleprotect.js

```diff
@@ -1159,7 +1214,11 @@ Twinkle.protect.callback.evaluate = function twinkleprotectCallbackEvaluate(e) {
 					typename = 'create protection';
 					break;
 				case 'unprotect':
-					/* falls through */
+					var admins = $.map(Twinkle.protect.currentProtectionLevels, function(pl) { return pl.admin ? 'User:' + pl.admin : null; });
+					if (admins.length && !confirm('Have you attempted to contact the protecting admins (' + $.unique(admins).join(', ') + ') first?' )) {
```

Comment:
Thoughts about doing a whole thing here for admin vs admins, but most of the good functions are in ES6 and it's just not really worth it.